server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerMetrics.java

```diff
@@ -46,6 +51,10 @@ public void registerMetrics(MeterRegistry registry) {
         .description("Number of queued major compactions").register(registry);
     Gauge.builder(METRICS_TSERVER_MINC_RUNNING, util, TabletServerMetricsUtil::getMinorCompactions)
         .description("Number of active minor compactions").register(registry);
+    Gauge
+        .builder(METRICS_TSERVER_MAJC_STUCK, util,
```

Comment:
If this is separate from Compactor METRICS_TSERVER_MAJC_STUCK and is the sum of stuck counts, then maybe named METRICS_TSERVER_MAJC_STUCK_COUNT or METRICS_TSERVER_MAJC_STUCK_TOTAL?  (I don't recall suggested naming conventions)

Reply:
They are the same metric, just being reported from different processes (the process name is in a tag). `METRICS_TSERVER_MAJC_STUCK` is just the variable name, the metric being reported is `metrics.tserver.majc.stuck`.